deploy/eks/action.yml

```diff
@@ -1,4 +1,4 @@
-name: 'eks-deploy'
+name: "eks-deploy"
```

Comment:
nit: Do we gain anything by converting these to double quotes or is it just personal preference? 
I was wondering when to use `""` vs `''` vs none in CI yml files in general 
(see `description:` fields below for an example that doesn't have any quotes)

No need to action on this, it is more for my education.